.github/workflows/update_gbs_cache.yml

```diff
@@ -1,4 +1,4 @@
-name: Update GBS cache once a day + nntrainer gbs build test
+name: daily build
```

Comment:
Please add a human readable title here... as long as this doesn't include all daily builds.

Reply:
The reason why I changed the title is that the name displayed on the badge is too long.
daily build result badge generated by Github Action: https://github.com/nnstreamer/nnstreamer/actions/workflows/update_gbs_cache.yml/badge.svg

This will revert this change and a new script will be added to create badges for each architecture.